vue-app/src/components/AddToCartButton.vue

```diff
@@ -135,6 +139,11 @@ export default class AddToCartButton extends Vue {
   toggleCartPanel() {
     this.$store.commit(TOGGLE_SHOW_CART_PANEL, true)
   }
+
+  get tokenLogo(): string {
+    const { nativeTokenSymbol } = this.$store.state.currentRound
+    return getTokenLogo(nativeTokenSymbol)
+  }
```

Comment:
Nice to have: to keep things consistent and easier to find, I would recommend having an order of declaration inside the components. Grouping them.

```ts
// 1. props
@Prop() prop

// 2. properties
property: string = ""

// 3. getters
get tokenLogo(): string {}

// 4. methods
newMethod(): void {}
```

This would apply for all the components/views.